config/packages/test/monolog.yaml

```diff
@@ -0,0 +1,8 @@
+monolog:
+    handlers:
+        main:
+            channels:
+                - '!event'
+            level: debug
+            path: '%kernel.logs_dir%/%kernel.environment%.log'
```

Comment:
if this remains inside the container, it's not that useful? But it's a safe default

Reply:
Shouldn't really be needed, and using `stderr` makes PHPUnit very noisy as the outputs are mixed up. Other options is to use a null logger, but no chance of seeing the output that way.